src/database/queries/resources.js

```diff
@@ -57,20 +57,29 @@ exports.findAll = async (filter, attributes = {}) => {
 		return error
 	}
 }
-exports.resourceList = async (filter, attributes = {}, sort = common.CREATED_AT, page = 1, limit = common.LIMIT) => {
+exports.resourceList = async (
+	filter,
+	attributes = {},
+	sort = { sort_by: common.CREATED_AT, order: common.SORT_DESC },
```

Comment:
@adithyadinesh0412 i think we don't need to add by default sort. Of sort is there then do sorting